src/Core/Query/Helper.php

```diff
@@ -172,15 +172,10 @@ public function formatDate($input)
 
         // handle the filtered input
         if ($input) {
-            // when we get here the input is always a datetime object
+            // when we get here $input is always a DateTime or DateTimeImmutable object
```

Comment:
I know this is nitpicking, but `$input` when we get here is no longer identical to "the input" that was passed as a parameter to the method.